.travis-push.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+setup_git() {
+  git config --global user.email "ci@coursier.iMadeThisUp"
```

Comment:
What email do you want here?

Reply:
I usually setup an invalid user in that kind of code, like `Travis-CI <invalid@travis-ci.com>`

Reply:
Sounds good. This is what I'll add in then.